apps/meteor/client/views/admin/moderation/ModerationConsoleTableRow.tsx

```diff
@@ -22,6 +22,13 @@ const ModerationConsoleTableRow = ({ report, onClick, isDesktopOrLarger }: Moder
 		return room.fname || room.name;
 	});
 
+	const normalizeMessage = () => {
+		if (message.startsWith('[ ](')) {
```

Comment:
Don't we have the whole message available here? If so, would it be better to use the existing `isQuoteAttachment` instead of checking the string? What if someone sends this exact text in the message?